### ISSUE TITLE
Widescreen monitor usb-c or hdmi

### ISSUE THREAD

**Issue by MrBlamz** [daliansky/XiaoMi-Pro-Hackintosh#411]
I searched everywhere and could no find anything. I am thinking about purchasing the enhanced edition and wanted to know if anyone here knows if it works in 3440x1440p resolution.
Thanks in advance.

**Comment by stevezhengshiqi** [comment]
Please clearly specify your problem. Is your HDMI not working? What is the resolution of your display?

**Comment by MrBlamz** [comment]
> Please clearly specify your problem. Is your HDMI not working? What is the resolution of your display?

Im looking to buy the enhanced version of xiaomi mi notebook pro and i wanted to know if someone knows if it works on 3440x1440p since it is the only monitor i have. I dont physically own it yet.

**Comment by almenjonatan** [comment]
I can get 3440x1440p with usb-c on my Dell 49 inch 5140x1440, but I cannot get my own native resolution.

**Comment by MrBlamz** [comment]
> I can get 3440x1440p with usb-c on my Dell 49 inch 5140x1440, but I cannot get my own native resolution.

Do you have the enhanced edition with the 10 gen CPU?

**Comment by almenjonatan** [comment]
> > I can get 3440x1440p with usb-c on my Dell 49 inch 5140x1440, but I cannot get my own native resolution.
> 
> Do you have the enhanced edition with the 10 gen CPU?

Nope 8th gen, I7 8550U

**Comment by almenjonatan** [comment]
Sorry, it was only up to 3k by 1080! I'm still wondering if anyone got 4k to work

**Comment by stevezhengshiqi** [comment]
@almenjonatan 
https://github.com/daliansky/XiaoMi-Pro-Hackintosh/issues/117
https://github.com/daliansky/XiaoMi-Pro-Hackintosh/tree/master/BIOS/DVMT_and_0xE2_fix

**Comment by almenjonatan** [comment]
@stevezhengshiqi Yeah, I've tried those, still same, although this seems to be an apple issue.

https://discussions.apple.com/thread/8641919

my monitor, displays at 3840x1080 in the display preference tab. It's kinda bad they don't have a fix for this.
